src/test/kotlin/io/libp2p/spike/Spike.kt.ignore

```diff
@@ -0,0 +1,85 @@
+package io.libp2p.spike
```

Comment:
I think this file's been included by accident?

Reply:
Thanks, removed!